lib/yubihsm_usb.c

```diff
@@ -74,7 +75,7 @@ static yh_rc backend_send_msg(yh_backend *connection, Msg *msg, Msg *response,
                               const char *identifier) {
   int32_t trf_len = msg->st.len + 3;
   yh_rc ret = YHR_GENERIC_ERROR;
-  unsigned long read_len;
+  unsigned long read_len = 0;
```

Comment:
Is this needed?

Reply:
MSVC gave a warning because read_len was not initialized. Do you think it's better to put it back?

Reply:
if it warns without it leave it there. I don't think it's actually true but doesn't matter.